aspnet/security/authentication/cookieauthentication.rst

```diff
@@ -0,0 +1,162 @@
+.. _security-authentication-cookie-middleware:
+
+Using Cookie Middleware without ASP.NET Identity
+================================================
+
+ASP.NET v5 provides cookie :ref:`middleware <fundamentals-middleware>` which serializes a user principal into an encrypted cookie and then, on subsequent requests, validates the cookie, recreates the principal and assigns it to the ``User`` property on ``HttpContext``. If you want to provide your on login screens and user databases you can use the cookie middleware as a standalone feature.
+
+.. _security-authentication-cookie-middleware-configuring:
+
+Adding and configuring
+----------------------
+
+The first step is adding the cookie middleware to your application. First use nuget to add the ``Microsoft.AspNet.Authentication.Cookies`` package. Then add the following lines to the ``Configure`` method in your ``Startup.cs`` file;
+
+.. code-block:: c#
+
+ app.UseCookieAuthentication(options =>
+ {
+     options.AuthenticationScheme = "MyCookieMiddlewareInstance";
+     options.LoginPath = new PathString("/Account/Unauthorized/");
+     options.AccessDeniedPath = new PathString("/Account/Forbidden/");
+     options.AutomaticAuthenticate = true;
+     options.AutomaticChallenge = true;
+ });
+
+The code snippet above configures a few options;
+
+* AuthenticationScheme - this is a value by which the middleware is known. This is useful when there are multiple instances of middleware and you want to :ref:`limit authorization to one instance <security-authorization-limiting-by-scheme>`.
+* LoginPath - this is the relative path requests will be redirected to when a user attempts to access a resource but has not been authenticated.
+* AccessDeniedPath - this is the relative path requests will be redirected to when a user attempts to access a resource but does not pass any :ref:`authorization policies <security-authorization-policies-based>` for that resource.
+* AutomaticAuthenticate - this flag indicates that the middleware should run on every request and attempt to validate and reconstruct any serialized principal it created.
```

Comment:
I bet 50$ that at least 90% of people will never realize the cookie middleware is automatically invoked for their Web API calls (since it's now the default mode).